### PR TITLE
Jira-service-desk branch to dev

### PR DESCRIPTION
**Pull request by ometelytsia**

### CODE REVIEW

File: app/jsm.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      - jpgc-wsc=0.3
+      - tilln-sshmon=1.0
+      - jpgc-synthesis=2.2
+    system-properties:
+      server.rmi.ssl.disable: true
+      java.rmi.server.hostname: localhost
+      httpsampler.ignore_failed_embedded_resources: "true"
+  selenium:
+    chromedriver:
+      version: "87.0.4280.88" # Supports Chrome version 87. You can refer to http://chromedriver.chromium.org/downloads
```
Comment (mmizin) [inline app/jsm.yml:155]: We have a new ChromeDriver 88.0.4324.27

File: docs/dc-apps-performance-toolkit-user-guide-jira.md [hunk 2]
```diff
@@ -882,12 +894,12 @@ To receive scalability benchmark results for four-node Jira DC with app-specific
...
    ``` bash
-    cd dc-app-performance-toolkit
-    docker run --shm-size=4g  -v "$PWD:/dc-app-performance-toolkit" atlassian/dcapt jira.yml
+   cd dc-app-performance-toolkit
+   docker run --shm-size=4g --pull="always" -v "$PWD:/dc-app-performance-toolkit" atlassian/dcapt jira.yml
    ```  
 
 {{% note %}}
-Review `results_summary.log` file under artifacts dir location. Make sure that overall status is `OK` before moving to the next steps.
+Review `results_summary.log` file under artifacts dir location. Make sure that overall status is `OK` before moving to the next steps. For enterprise-scale environment run results acceptable success rate for actions is 95% and above.
```
Comment (mmizin) [inline docs/dc-apps-performance-toolkit-user-guide-jira.md:902]: How do you think about such a variant as "For an enterprise-scale environment, the acceptable actions success rate is 95% and above." ?

**Review by mmizin — APPROVED** [review]

File: docs/dc-apps-performance-toolkit-user-guide-jira.md (hunk 2 above)
Comment (ometelytsia) [reply docs/dc-apps-performance-toolkit-user-guide-jira.md]: Improved wording. Thanks.

File: app/jira.yml [hunk 3]
```diff
@@ -113,7 +113,7 @@ modules:
       httpsampler.ignore_failed_embedded_resources: "true"
   selenium:
     chromedriver:
-      version: "87.0.4280.20" # Supports Chrome version 87. You can refer to http://chromedriver.chromium.org/downloads
+      version: "87.0.4280.88" # Supports Chrome version 87. You can refer to http://chromedriver.chromium.org/downloads
```
[the remark this answers is not in the document]
Comment (ometelytsia) [reply app/jira.yml:116]: My local Chrome is up to date and still has 87 version.
The same CI suggests that 87.0.4280.88 is the latest chrome driver for 87 version.
Do you have Chrome 88 already?

**Review by astashys — APPROVED** [review]

**Pull request merged**